src/lavinmq/launcher.cr

```diff
@@ -89,6 +91,26 @@ module LavinMQ
       Log.info &.emit("logger settings", level: @config.log_level.to_s, target: target)
     end
 
+    private def setup_log_exchange
+      queue_name = "logstream"
+      exchange_name = "amq.lavinmq.log"
+      vhost = @amqp_server.vhosts["/"]
+      vhost.declare_exchange(exchange_name, "topic", true, true, true)
+      vhost.declare_queue(queue_name, true, false)
+      vhost.bind_queue(queue_name, exchange_name, "#")
```

Comment:
No need to declare the queue, that's up to the user to do.

```suggestion
      exchange_name = "amq.lavinmq.log"
      vhost = @amqp_server.vhosts["/"]
      vhost.declare_exchange(exchange_name, "topic", true, true, true)
```

Reply:
> We don't allow binding to internal exchanges

Hmm, that's only in the mgmt interface though. I can bind to it with a client, so that should be fine.

Reply:
That's a bug. Users shouldnt be able to publish to internal exchanges, but should be able to bind to them